src/styles/reset.scss

```diff
@@ -0,0 +1,133 @@
+html,
```

Comment:
Interesting! I haven't seen this before but I see the utility after Googling:
https://meyerweb.com/eric/tools/css/reset/